src/DevAdventCalendarCompetition/DevAdventCalendarCompetition.Repository/HomeRepository.cs

```diff
@@ -1,10 +1,13 @@
-﻿using System.Collections.Generic;
+﻿using System;
+using System.Collections.Generic;
 using System.Linq;
+using System.Linq.Expressions;
```

Comment:
please add migration to set NULL by default for all Points and Places in Results table